### PR TITLE
Update ediscovery-attorney-privilege-detection.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note that the Attorney list cannot be viewed or downloaded after uploading. So it is recommended to keep the file for reference

Fixes#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/11231

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/ediscovery-attorney-privilege-detection.md [hunk 1]
```diff
@@ -81,6 +81,9 @@ To upload an attorney list for use by the attorney-client privilege detection mo
 
 4. Select **Save** to upload the attorney list.
 
+> [!NOTE]
+> Once the attorney CSV file has been uploaded, it is not possible to view or download the attorney list. It is recommended to keep the CSV file for reference.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ediscovery-attorney-privilege-detection.md]:
```suggestion
> Once the attorney CSV file has been uploaded, it is not possible to view or download the attorney list. We recommend keeping the CSV file for reference.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@robmazz Gentle reminder. Thanks

**Comment by yogkumgit** [comment]
Closing as per the comment in #11231

**Pull request closed without merging**